crates/router/src/core/admin.rs

```diff
@@ -470,6 +470,10 @@ pub async fn merchant_account_delete(
         .delete_merchant_account_by_merchant_id(&merchant_id)
         .await
         .to_not_found_response(errors::ApiErrorResponse::MerchantAccountNotFound)?;
+    let _is_deleted = db
+        .delete_merchant_key_store_by_merchant_id(&merchant_id)
+        .await
+        .to_not_found_response(errors::ApiErrorResponse::MerchantAccountNotFound)?;
```

Comment:
I was not sure how to use `_is_deleted` and if I should delete the `merchant_key_store` after or before deleting the `merchant_account`

Reply:
Probably You can do 
```rust
   let is_merchant_account_deleted = db
        .delete_merchant_account_by_merchant_id(&merchant_id)
        .await
        .to_not_found_response(errors::ApiErrorResponse::MerchantAccountNotFound)?;
    let is_merchant_key_store_deleted = db
        .delete_merchant_key_store_by_merchant_id(&merchant_id)
        .await
        .to_not_found_response(errors::ApiErrorResponse::MerchantKeyStoreNotFound)?;
   
   return is_merchant_account_deleted && is_merchant_key_store_deleted
```

Reply:
`errors::ApiErrorResponse::MerchantKeyStoreNotFound` does not exist, how to proceed? Should I implement?

Reply:
Please create it in the ApiResponse

Reply:
Done.